Mirnawong1 faq addition

runleonarun opened the pull request:
## Description & motivation
<!---
Describe your changes, and why you're making them. Is this linked to an open
issue, a pull request on dbt core, etc?
-->

## To-do before merge
<!---
(Optional -- remove this section if not needed)
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Change the base branch
- [ ] Ensure PR #56 is merged
-->

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](../contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](../website/docs/docs/guides/migration-guide)

## Checklist
If you added new pages (delete if not applicable):
- [x] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

runleonarun commented:
@JKarlavige, this builds locally, but the Netlify build fails. I want to be able to reference the `id` instead of the path to the file or filename when we use faqs inline for example `<FAQ src="dbt-specific-jinja" />` (not sure what it's using now). I wasn't sure how to make that happen for FAQs.  I'm assuming this is what's causing the build errors?

Hi @mirnawong1 and @sophiad96! I just wanted to capture some of the changes I made so you can understand the fixes I made, and feel free to ask questions about them!
- Added directories that will contain the content for each subcategory (like "models" and "troubleshooting")
- Made all of the content in each directory automatically generated, so when you add content to the folder, and it will show up in the category sidebar.
- I didn't move all of the content into directories because I wasn't sure where it all went.
- You no longer have to edit the `sidebar.js` unless you want to add another category (like "security" Note:  I didn't add "security" because you can't have an empty folder so I didn't want it to break.)
- Shortened some titles using a `sidebar_label: 'How to name models directory'` or by making the title shorter. (see the Models and Tests directories for examples) when you specify a sidebar_label, it overrides the title in the left bar.
- Fixed some of the markdown files to remove the lingering code `</File>`
- Fixed links that no longer worked.
- I also started adding ids to the FAQs.md files in the header. We might need these in each FAQ depending on Jason's answer above.
- 
You can see the changes I made [here](https://github.com/dbt-labs/docs.getdbt.com/pull/1404/files/f6b941a3eb7e0f5db8256beaf54846cb2c778ba1..a894da9ddb17d56c9ae3afd8f8acd395dfe4bd65).

JKarlavige commented:
@runleonarun you are correct, the FAQs component is referencing the filename with the following code `const file = require('../../../docs/faqs/' + src + '.md')`

Since the faqs are moved into subdirectories, the path above is no longer accurate and causing the build to fail, as it is unable to find those files. We will have to update the FAQs component to support this. 

So to confirm, this should be adjusted to get the FAQ by the `id` in the frontmatter instead of the filename? I will put a ticket into our asana board to get that adjusted. I will open a separate PR to adjust that, and will set it up so it supports both the `id` and filename, so it will work with the current production code, as well as this code once it's merged in. I _think_ that will be possible.

runleonarun commented:
> So to confirm, this should be adjusted to get the FAQ by the id in the frontmatter instead of the filename? I will put a ticket into our asana board to get that adjusted. I will open a separate PR to adjust that, and will set it up so it supports both the id and filename, so it will work with the current production code, as well as this code once it's merged in. I think that will be possible.

Yep! if that fixes the errors, then I think that's the preferred path. I'd also like to fallback on the filename if no id is provided. Just in case.

mirnawong1 committed:
Update documenting-macros.md

mirnawong1 committed:
Update long-descriptions.md

mirnawong1 committed:
Update sharing-documentation.md

mirnawong1 committed:
Update beta-release.md

mirnawong1 committed:
Update diff-database-environment.md

mirnawong1 committed:
Update profile-env-vars.md

mirnawong1 committed:
Update diff-database-environment.md

mirnawong1 committed:
Update profile-name.md

mirnawong1 committed:
Update target-names.md

mirnawong1 committed:
Create git-revlist-error.md

mirnawong1 committed:
Delete git-revlist-error.md

mirnawong1 committed:
Update gitignore.md

mirnawong1 committed:
Create gitignore.md

mirnawong1 committed:
Update gitlab-authentication.md

mirnawong1 committed:
Create gitlab-authentication.md

mirnawong1 committed:
Update gitlab-selfhosted.md

mirnawong1 committed:
Update managed-repo.md

mirnawong1 committed:
Update run-on-pull.md

mirnawong1 committed:
Update jinja-whitespace.md

mirnawong1 committed:
Update quoting-column-names.md

mirnawong1 committed:
Update which-jinja-docs.md

mirnawong1 committed:
Update available-configurations.md

mirnawong1 committed:
Update available-materializations.md

mirnawong1 committed:
Update configurable-model-path.md

mirnawong1 committed:
Update create-a-schema.md

mirnawong1 committed:
Update create-dependencies.md

mirnawong1 committed:
Update insert-records.md

mirnawong1 committed:
Update model-custom-schemas.md

mirnawong1 committed:
Update reference-models-in-another-project.md

mirnawong1 committed:
Update removing-deleted-models.md

mirnawong1 committed:
Update run-downtime.md

mirnawong1 committed:
Update source-quotes.md

mirnawong1 committed:
Update specifying-column-types.md

mirnawong1 committed:
Update sql-dialect.md

mirnawong1 committed:
Update unique-model-names.md

mirnawong1 committed:
Update dbt-source-freshness.md

mirnawong1 committed:
Update debugging-jinja.md

mirnawong1 committed:
Update define-a-column-type.md

mirnawong1 committed:
Update docs-for-multiple-projects.md

mirnawong1 committed:
Update example-projects.md

mirnawong1 committed:
Update exclude-table-from-freshness.md

mirnawong1 committed:
Update multiple-resource-yml-files.md

mirnawong1 committed:
Update project-name.md

mirnawong1 committed:
Update properties-not-in-config.md

mirnawong1 committed:
Update properties-not-in-config.md

mirnawong1 committed:
Update resource-yml-name.md

mirnawong1 committed:
Update schema-yml-name.md

mirnawong1 committed:
Update separate-profile.md

mirnawong1 committed:
Update source-has-bad-name.md

mirnawong1 committed:
Update source-in-different-database.md

mirnawong1 committed:
Update structure-a-project.md

mirnawong1 committed:
Update which-materialization.md

mirnawong1 committed:
Create access_token_error.md

mirnawong1 committed:
Delete prerelease-docs.md

deleting per leona's meeting here: https://www.notion.so/dbtlabs/Leona-x-Mirna-f36937da0c0547149cb941f1a06859c3 as pre release docs are archived

mirnawong1 committed:
Delete cloud-vs-core.md

moving the 'what is the diff btw dbt core and cloud' section from faq to docs per meeting with Leona https://www.notion.so/dbtlabs/Leona-x-Mirna-f36937da0c0547149cb941f1a06859c3

mirnawong1 committed:
Update which-schema.md

mirnawong1 committed:
Update why-not-write-dml.md

mirnawong1 committed:
Update why-so-many-macros.md

mirnawong1 committed:
Update why-version-2.md

mirnawong1 committed:
Update yaml-file-extension.md

mirnawong1 committed:
Update checking-logs.md

mirnawong1 committed:
Update failed-prod-run.md

mirnawong1 committed:
Update failed-tests.md

mirnawong1 committed:
Update run-downstream-of-seed.md

mirnawong1 committed:
Update run-one-model.md

mirnawong1 committed:
Update run-one-snapshot.md

mirnawong1 committed:
Update running-models-downstream-of-source.md

mirnawong1 committed:
Update snapshot-frequency.md

mirnawong1 committed:
Update build-one-seed.md

mirnawong1 committed:
Update full-refresh-seed.md

mirnawong1 committed:
Update leading-zeros-in-seed.md

mirnawong1 committed:
Update load-raw-data-with-seed.md

mirnawong1 committed:
Update seed-custom-schemas.md

mirnawong1 committed:
Update seed-datatypes.md

mirnawong1 committed:
Update seed-hooks.md

mirnawong1 committed:
Update snapshot-hooks.md

mirnawong1 committed:
Update snapshot-schema-changes.md

mirnawong1 committed:
Update snapshot-target-schema.md

mirnawong1 committed:
Update snapshotting-freshness-for-one-source.md

mirnawong1 committed:
Update available-tests.md

mirnawong1 committed:
Update configurable-data-path.md

mirnawong1 committed:
Update configurable-data-test-path.md

mirnawong1 committed:
Update configurable-data-path.md

mirnawong1 committed:
Update custom-test-thresholds.md

mirnawong1 committed:
Update multiple-test-files.md

mirnawong1 committed:
Update recommended-tests.md

mirnawong1 committed:
Update test-one-model.md

mirnawong1 committed:
Update test-sources.md

mirnawong1 committed:
Update testing-seeds.md

mirnawong1 committed:
Update testing-sources.md

mirnawong1 committed:
Update testing-sources.md

mirnawong1 committed:
Delete test-sources.md

duplicate of https://github.com/dbt-labs/docs.getdbt.com/blob/mirnawong1-faq-addition/website/docs/faqs/tests/testing-sources.md

mirnawong1 committed:
Update uniqueness-two-columns.md

mirnawong1 committed:
Update when-to-test.md

mirnawong1 committed:
Update access-gdrive-credential.md

mirnawong1 committed:
Update access_token_error.md

mirnawong1 committed:
Update dispatch-could-not-find-package.md

mirnawong1 committed:
Update nonetype-ide-error.md

mirnawong1 committed:
Update partial-parsing-error.md

mirnawong1 committed:
Update runtime-error-could-not-find-profile.md

mirnawong1 committed:
Update runtime-packages.yml.md

mirnawong1 committed:
Update sql-errors.md

mirnawong1 committed:
Update unused-model-configurations.md

mirnawong1 committed:
Update bq-impersonate-service-account-setup.md

mirnawong1 committed:
Update bq-impersonate-service-account-why.md

mirnawong1 committed:
Update connecting-to-two-dbs-not-allowed.md

mirnawong1 committed:
Update database-privileges.md

mirnawong1 committed:
Update loading-data.md

mirnawong1 committed:
Update sample-profiles.md

mirnawong1 committed:
Update sidebars.js

added add'l folders and order to sidebar.js

mirnawong1 committed:
Create google-cloud-repo.md

Commit:
Merge branch 'current' into mirnawong1-faq-addition

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

Commit:
making quotes not curly and adjustiung the sidebar code

Commit:
Merge branch 'mirnawong1-faq-addition' of github.com:dbt-labs/docs.getdbt.com into mirnawong1-faq-addition

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

runleonarun commented on website/src/pages/index.js:
```diff
@@ -261,7 +261,7 @@ function Home() {
                           <Link
                             className="button button--primary"
-                            to={useBaseUrl('/faqs/example-projects')}>
+                            to={useBaseUrl('/faqs/project/example-projects')}>
```
```suggestion
                            to={useBaseUrl('/faqs/example-projects')}>
```

runleonarun committed:
Update website/src/pages/index.js

Commit:
fixing build errors

Commit:
moving faqs under Docs

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

mirnawong1 committed:
Create landing_page.md

create a new faq landing page to greet users landing into mvp faq site here: https://docs.getdbt.com/faqs/all

mirnawong1 committed:
Delete landing_page.md

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

Commit:
correction to see if folder changes are also pushed

Commit:
changing directory case

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

Commit:
fixing links

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

Commit:
makign capitlalization changes

mirnawong1 committed:
Update sidebars.js

added landing page blurb

mirnawong1 committed:
Rename website/docs/faqs/Accounts1/change-billing.md to website/docs/faqs/Accounts/change-billing.md

mirnawong1 committed:
Delete website/docs/faqs/Accounts1 directory

mirnawong1 committed:
Create configurable-snapshot-path.md

mirnawong1 committed:
Create dbt-specific-jinja.md

mirnawong1 committed:
Create git-account-in-use.md

mirnawong1 committed:
Create payment-accepted.md

mirnawong1 committed:
Create slack.md

mirnawong1 committed:
Rename website/docs/faqs/Core1/install-pip-best-practices.md to website/docs/faqs/Core/install-pip-best-practices.md

mirnawong1 committed:
Rename website/docs/faqs/Core1/install-pip-os-prereqs.md to website/docs/faqs/Core/install-pip-os-prereqs.md

mirnawong1 committed:
Rename website/docs/faqs/Core1/install-python-compatibility.md to website/docs/faqs/Core/install-python-compatibility.md

mirnawong1 committed:
Rename website/docs/faqs/Docs1/document-all-columns.md to website/docs/faqs/Docs/document-all-columns.md

mirnawong1 committed:
Rename website/docs/faqs/Docs1/document-other-resources.md to website/docs/faqs/Docs/document-other-resources.md

mirnawong1 committed:
Rename website/docs/faqs/Docs1/documenting-macros.md to website/docs/faqs/Docs/documenting-macros.md

mirnawong1 committed:
Rename website/docs/faqs/Docs1/long-descriptions.md to website/docs/faqs/Docs/long-descriptions.md

mirnawong1 committed:
Rename website/docs/faqs/Docs1/sharing-documentation.md to website/docs/faqs/Docs/sharing-documentation.md

mirnawong1 committed:
Rename website/docs/faqs/Environments1/beta-release.md to website/docs/faqs/Environments/beta-release.md

mirnawong1 committed:
Rename website/docs/faqs/Environments1/diff-database-environment.md to website/docs/faqs/Environments/diff-database-environment.md

mirnawong1 committed:
Rename website/docs/faqs/Environments1/profile-env-vars.md to website/docs/faqs/Environments/profile-env-vars.md

mirnawong1 committed:
Rename website/docs/faqs/Environments1/profile-name.md to website/docs/faqs/Environments/profile-name.md

mirnawong1 committed:
Rename website/docs/faqs/Environments1/target-names.md to website/docs/faqs/Environments/target-names.md

mirnawong1 committed:
Rename website/docs/faqs/Git1/gitignore.md to website/docs/faqs/Git/gitignore.md

mirnawong1 committed:
Rename website/docs/faqs/Git1/gitlab-authentication.md to website/docs/faqs/Git/gitlab-authentication.md

mirnawong1 committed:
Rename website/docs/faqs/Git1/google-cloud-repo.md to website/docs/faqs/Git/google-cloud-repo.md

mirnawong1 committed:
Rename website/docs/faqs/Git1/managed-repo.md to website/docs/faqs/Git/managed-repo.md

mirnawong1 committed:
Rename website/docs/faqs/Git1/run-on-pull.md to website/docs/faqs/Git/run-on-pull.md

mirnawong1 committed:
Rename website/docs/faqs/Git1/gitlab-selfhosted.md to website/docs/faqs/Git/gitlab-selfhosted.md

mirnawong1 committed:
Rename website/docs/faqs/Jinja1/jinja-whitespace.md to website/docs/faqs/Jinja/jinja-whitespace.md

mirnawong1 committed:
Rename website/docs/faqs/Jinja1/quoting-column-names.md to website/docs/faqs/Jinja/quoting-column-names.md

mirnawong1 committed:
Rename website/docs/faqs/Jinja1/which-jinja-docs.md to website/docs/faqs/Jinja/which-jinja-docs.md

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

Commit:
Merge branch 'current' into mirnawong1-faq-addition

Commit:
Merge branch 'mirnawong1-faq-addition' of github.com:dbt-labs/docs.getdbt.com into mirnawong1-faq-addition

runleonarun commented on website/docs/docs/available-adapters.md:
```diff
@@ -5,7 +5,7 @@ id: "available-adapters"
 dbt connects to and runs SQL against your database, warehouse, platform, or query engine. It works by using a dedicated **adapter** for each technology. All the adapters listed below are open source and free to use, just like dbt.
 
-If you have a new adapter, please add it to this list via a pull request! See [Documenting your adapter](/website/docs/docs/contributing/documenting-a-new-adapter.md)  for more information.
+If you have a new adapter, please add it to this list via a pull request! See [Documenting your adapter](/docs/contributing/documenting-a-new-adapter.md)  for more information.
```
```suggestion
If you have a new adapter, please add it to this list using a pull request! See [Documenting your adapter](/docs/contributing/documenting-a-new-adapter.md)  for more information.
```

runleonarun committed:
Update website/docs/docs/available-adapters.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/available-configurations.md to website/docs/faqs/Models/available-configurations.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/available-materializations.md to website/docs/faqs/Models/available-materializations.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/configurable-model-path.md to website/docs/faqs/Models/configurable-model-path.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/create-a-schema.md to website/docs/faqs/Models/create-a-schema.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/create-dependencies.md to website/docs/faqs/Models/create-dependencies.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/insert-records.md to website/docs/faqs/Models/insert-records.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/model-custom-schemas.md to website/docs/faqs/Models/model-custom-schemas.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/reference-models-in-another-project.md to website/docs/faqs/Models/reference-models-in-another-project.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/removing-deleted-models.md to website/docs/faqs/Models/removing-deleted-models.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/run-downtime.md to website/docs/faqs/Models/run-downtime.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/source-quotes.md to website/docs/faqs/Models/source-quotes.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/specifying-column-types.md to website/docs/faqs/Models/specifying-column-types.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/sql-dialect.md to website/docs/faqs/Models/sql-dialect.md

mirnawong1 committed:
Rename website/docs/faqs/Models1/unique-model-names.md to website/docs/faqs/Models/unique-model-names.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/dbt-source-freshness.md to website/docs/faqs/Project/dbt-source-freshness.md

renaming files manually using new mirnawong1-faq-addition2 branch to avoid the '1k lines exceed error'

mirnawong1 committed:
Merge pull request #1546 from dbt-labs/mirnawong1-faq-addition2

Rename website/docs/faqs/Project1/dbt-source-freshness.md to website/…

mirnawong1 committed:
Rename website/docs/faqs/Project1/debugging-jinja.md to website/docs/faqs/Project/debugging-jinja.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/define-a-column-type.md to website/docs/faqs/Project/define-a-column-type.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/docs-for-multiple-projects.md to website/docs/faqs/Project/docs-for-multiple-projects.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/example-projects.md to website/docs/faqs/Project/example-projects.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/exclude-table-from-freshness.md to website/docs/faqs/Project/exclude-table-from-freshness.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/multiple-resource-yml-files.md to website/docs/faqs/Project/multiple-resource-yml-files.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/project-name.md to website/docs/faqs/Project/project-name.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/properties-not-in-config.md to website/docs/faqs/Project/properties-not-in-config.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/resource-yml-name.md to website/docs/faqs/Project/resource-yml-name.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/schema-yml-name.md to website/docs/faqs/Project/schema-yml-name.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/separate-profile.md to website/docs/faqs/Project/separate-profile.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/source-has-bad-name.md to website/docs/faqs/Project/source-has-bad-name.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/source-in-different-database.md to website/docs/faqs/Project/source-in-different-database.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/structure-a-project.md to website/docs/faqs/Project/structure-a-project.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/which-materialization.md to website/docs/faqs/Project/which-materialization.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/which-schema.md to website/docs/faqs/Project/which-schema.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/why-not-write-dml.md to website/docs/faqs/Project/why-not-write-dml.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/why-so-many-macros.md to website/docs/faqs/Project/why-so-many-macros.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/why-version-2.md to website/docs/faqs/Project/why-version-2.md

mirnawong1 committed:
Rename website/docs/faqs/Project1/yaml-file-extension.md to website/docs/faqs/Project/yaml-file-extension.md

mirnawong1 committed:
Rename website/docs/faqs/Runs1/checking-logs.md to website/docs/faqs/Runs/checking-logs.md

mirnawong1 committed:
Rename website/docs/faqs/Runs1/failed-prod-run.md to website/docs/faqs/Runs/failed-prod-run.md

mirnawong1 committed:
Rename website/docs/faqs/Runs1/failed-tests.md to website/docs/faqs/Runs/failed-tests.md

mirnawong1 committed:
Rename website/docs/faqs/Runs1/run-downstream-of-seed.md to website/docs/faqs/Runs/run-downstream-of-seed.md

mirnawong1 committed:
Rename website/docs/faqs/Runs1/run-one-model.md to website/docs/faqs/Runs/run-one-model.md

mirnawong1 committed:
Rename website/docs/faqs/Runs1/run-one-snapshot.md to website/docs/faqs/Runs/run-one-snapshot.md

mirnawong1 committed:
Rename website/docs/faqs/Runs1/running-models-downstream-of-source.md to website/docs/faqs/Runs/running-models-downstream-of-source.md

mirnawong1 committed:
Rename website/docs/faqs/Runs1/snapshot-frequency.md to website/docs/faqs/Runs/snapshot-frequency.md

mirnawong1 committed:
Rename website/docs/faqs/Seeds1/build-one-seed.md to website/docs/faqs/Seeds/build-one-seed.md

mirnawong1 committed:
Rename website/docs/faqs/Seeds1/full-refresh-seed.md to website/docs/faqs/Seeds/full-refresh-seed.md

mirnawong1 committed:
Rename website/docs/faqs/Seeds1/leading-zeros-in-seed.md to website/docs/faqs/Seeds/leading-zeros-in-seed.md

mirnawong1 committed:
Rename website/docs/faqs/Seeds1/load-raw-data-with-seed.md to website/docs/faqs/Seeds/load-raw-data-with-seed.md

mirnawong1 committed:
Rename website/docs/faqs/Seeds1/seed-custom-schemas.md to website/docs/faqs/Seeds/seed-custom-schemas.md

mirnawong1 committed:
Rename website/docs/faqs/Seeds1/seed-datatypes.md to website/docs/faqs/Seeds/seed-datatypes.md

mirnawong1 committed:
Rename website/docs/faqs/Seeds1/seed-hooks.md to website/docs/faqs/Seeds/seed-hooks.md

mirnawong1 committed:
Rename website/docs/faqs/Snapshots1/snapshot-hooks.md to website/docs/faqs/Snapshots/snapshot-hooks.md

mirnawong1 committed:
Rename website/docs/faqs/Snapshots1/snapshot-schema-changes.md to website/docs/faqs/Snapshots/snapshot-schema-changes.md

mirnawong1 committed:
Rename website/docs/faqs/Snapshots1/snapshotting-freshness-for-one-source.md to website/docs/faqs/Snapshots/snapshotting-freshness-for-one-source.md

mirnawong1 committed:
Rename website/docs/faqs/Snapshots1/snapshot-target-schema.md to website/docs/faqs/Snapshots/snapshot-target-schema.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/available-tests.md to website/docs/faqs/Tests/available-tests.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/configurable-data-path.md to website/docs/faqs/Tests/configurable-data-path.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/configurable-data-test-path.md to website/docs/faqs/Tests/configurable-data-test-path.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/custom-test-thresholds.md to website/docs/faqs/Tests/custom-test-thresholds.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/multiple-test-files.md to website/docs/faqs/Tests/multiple-test-files.m

mirnawong1 committed:
Rename website/docs/faqs/Tests1/recommended-tests.md to website/docs/faqs/Tests/recommended-tests.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/test-one-model.md to website/docs/faqs/Tests/test-one-model.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/testing-seeds.md to website/docs/faqs/Tests/testing-seeds.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/testing-sources.md to website/docs/faqs/Tests/testing-sources.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/uniqueness-two-columns.md to website/docs/faqs/Tests/uniqueness-two-columns.md

mirnawong1 committed:
Rename website/docs/faqs/Tests1/when-to-test.md to website/docs/faqs/Tests/when-to-test.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/access-gdrive-credential.md to website/docs/faqs/Troubleshooting/access-gdrive-credential.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/access_token_error.md to website/docs/faqs/Troubleshooting/access_token_error.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/dispatch-could-not-find-package.md to website/docs/faqs/Troubleshooting/dispatch-could-not-find-package.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/git-revlist-error.md to website/docs/faqs/Troubleshooting/git-revlist-error.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/gitignore.md to website/docs/faqs/Troubleshooting/gitignore.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/gitlab-authentication.md to website/docs/faqs/Troubleshooting/gitlab-authentication.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/nonetype-ide-error.md to website/docs/faqs/Troubleshooting/nonetype-ide-error.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/partial-parsing-error.md to website/docs/faqs/Troubleshooting/partial-parsing-error.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/runtime-error-could-not-find-profile.md to website/docs/faqs/Troubleshooting/runtime-error-could-not-find-profile.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/runtime-packages.yml.md to website/docs/faqs/Troubleshooting/runtime-packages.yml.md

mirnawong1 committed:
Update sql-errors.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/sql-errors.md to website/docs/faqs/Troubleshooting/sql-errors.md

mirnawong1 committed:
Rename website/docs/faqs/Troubleshooting1/unused-model-configurations.md to website/docs/faqs/Troubleshooting/unused-model-configurations.md

mirnawong1 committed:
Rename website/docs/faqs/Warehouse1/bq-impersonate-service-account-setup.md to website/docs/faqs/Warehouse/bq-impersonate-service-account-setup.md

mirnawong1 committed:
Rename website/docs/faqs/Warehouse1/bq-impersonate-service-account-why.md to website/docs/faqs/Warehouse/bq-impersonate-service-account-why.md

mirnawong1 committed:
Rename website/docs/faqs/Warehouse1/connecting-to-two-dbs-not-allowed.md to website/docs/faqs/Warehouse/connecting-to-two-dbs-not-allowed.md

mirnawong1 committed:
Rename website/docs/faqs/Warehouse1/database-privileges.md to website/docs/faqs/Warehouse/database-privileges.md

mirnawong1 committed:
Rename website/docs/faqs/Warehouse1/loading-data.md to website/docs/faqs/Warehouse/loading-data.md

mirnawong1 committed:
Rename website/docs/faqs/Warehouse1/sample-profiles.md to website/docs/faqs/Warehouse/sample-profiles.md

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

runleonarun committed:
Merge branch 'current' into mirnawong1-faq-addition

mirnawong1 reviewed (commented):
Approving review per Leona's feedback

mirnawong1 reviewed (approved):
Approving and merging pr
